components/automate-cs-nginx/habitat/plan.sh

```diff
@@ -47,7 +46,7 @@ pkg_upstream_url="https://www.chef.io/automate"
 # the other nginx instances in A2 at the moment:
 pkg_svc_user="root"
 
-pkg_scaffolding=chef/scaffolding-go
+pkg_scaffolding="${local_scaffolding_origin:-chef}/automate-scaffolding-go"
```

Comment:
its unclear why this works. yzl added some stuff to the packages.meta file that you have to set to true to say you have platform scaffolding

Reply:
it seems the meaning of not editing package.meta is that the service wont get external postgres info